WebApp/WebContent/resources/script/newInstrument.js

```diff
@@ -157,6 +156,18 @@ function updateColumnCount() {
   renderSampleFile();
 }
 
+
```

Comment:
Any particular reason for these gaps in the file?

Reply:
They are there for an unknown reason. Removed them now...